blocks/product-recommendations/product-recommendations.js

```diff
@@ -10,6 +10,7 @@ import { performCatalogServiceQuery } from '../../scripts/commerce.js';
 import '../../scripts/initializers/cart.js';
 import '../../scripts/initializers/wishlist.js';
 import { rootLink } from '../../scripts/scripts.js';
+import { tryGenerateAemAssetsOptimizedUrl } from '../../scripts/assets.js';
```

Comment:
Please revert these changes to `product-recommendations.js` so that this can be implemented in the drop-in instead.